packages/atomic/src/components/common/interface/interface-common.tsx

```diff
@@ -2,18 +2,158 @@ import type {SearchEngine} from '@coveo/headless';
 import {LogLevel} from '@coveo/headless';
 import type {RecommendationEngine} from '@coveo/headless/recommendation';
 import type {InsightEngine} from '@coveo/headless/insight';
-import {i18n} from 'i18next';
+import {i18n, TFunction} from 'i18next';
+import {ComponentInterface, h} from '@stencil/core';
+import {setCoveoGlobal} from '../../../global/environment';
+import {loadFocusVisiblePolyfill} from '../../../global/focus-visible';
+import {initi18n} from './i18n';
+import {HTMLStencilElement} from '@stencil/core/internal';
+import {loadDayjsLocale} from '../../../utils/dayjs-locales';
+import {InitializeEvent} from '../../../utils/initialization-utils';
+import {AnyBindings} from './bindings';
+import {
+  i18nBackendOptions,
+  i18nTranslationNamespace,
+} from '../../common/interface/i18n';
+import Backend from 'i18next-http-backend';
 
 export type AnyEngineType = SearchEngine | RecommendationEngine | InsightEngine;
```

Comment:
how about moving this to the `bindings` file to avoid the both-ways importing?

Reply:
Yes, great idea !